examples/openal/WavePlayer/WavePlayer.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net5.0</TargetFramework>
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Silk.NET.OpenAL" Version="2.3.0" />
```

Comment:
?

Reply:
Honestly not sure what we chose for the examples, should I instead reference directly?

Reply:
Yeah we do ProjectReferences (or should prefer to)